examples/cli/web-socket-url-cli/README.md

```diff
@@ -1,14 +1,14 @@
 # CLI: Web Socket URL
 
 ```console
-npx webpack serve --open-target --host 0.0.0.0 --web-socket-url <insert-host>:8080
+npx webpack serve --open-target --host 0.0.0.0 --client-web-socket-url <insert-host>:8080
```

Comment:
it should have `ws://` prefix, i.e. full valid URL

Reply:
fixed